tests/behat/features/ra_candidate2.feature

```diff
@@ -0,0 +1,30 @@
+Feature: A RAA manages ra candidates in the ra environment
+  In order to promote candidates
+  As a RAA
+  I must be able to promote and demote identities were I'm allowed to through the authorization config
+
+  Scenario: Provision an institution and a user to promote later on by an authorized institution
+    # Institution A
+    Given institution "institution-a.example.com" can "select_raa" from institution "institution-a.example.com"
+      And institution "institution-a.example.com" can "use_ra" from institution "institution-a.example.com"
+      And institution "institution-a.example.com" can "use_raa" from institution "institution-a.example.com"
+
+    # Institution V
+      And institution "institution-d.example.com" can "select_raa" from institution "institution-a.example.com"
+      And institution "institution-d.example.com" can "select_raa" from institution "institution-b.example.com"
```

Comment:
Institution b reference seems unrelated? Or would this clash with default configuration?

Reply:
With the current knowledge it indeed seems unrelated but initially it was impossible to conclude that.

Reply:
Allright, makes sense!